test/regression.bzl

```diff
@@ -1,3 +1,4 @@
+# -*- Python -*-
```

Comment:
Do we need to put this in every bzl file?  Why not just setup emacs to recognize the extension?

Reply:
I should probably do that :)

Reply:
I'm using
```
(defun auto-mode (expr mode)
  (setq auto-mode-alist (cons (cons expr mode)
			      auto-mode-alist)))

(auto-mode "\\.bazel$" 'python-mode)
(auto-mode "\\.bzl$" 'python-mode)
(auto-mode "BUILD$" 'python-mode)
(auto-mode "WORKSPACE$" 'python-mode)
```

Reply:
Thanks, now added to my `~/.emacs`

Reply:
Removed the `-*- Python -*-` and added a SPDX license identifier.